### PR TITLE
Add iM880B to readme

### PR DESCRIPTION
**Pull request by itziardelatorre**
Add iM880B to readme file.

@AnttiKauppila: could you please review it? Thanks!

### CODE REVIEW

**Comment by itziardelatorre** [comment]
@AnttiKauppila @adbridge would you please review?

**Comment by cmonr** [comment]
@ARMmbed/mbed-os-maintainers Someone mind taking a look? 

@itziardelatorre @adbridge is on vacation at the moment, and I think @AnttiKauppila might be as well?

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Here is a nonexhaustive list of boards and modules that we have tested with the
 - LTEK_FF1705.
 - Advantech Wise 1510.
 - ST B-L072Z-LRWAN1 LoRa®Discovery kit (with muRata radio chip).
+- IMST iM880B.
```
Comment (evedon) [inline README.md]: Could you create a different list to indicate that this board was tested, but now by Arm?

Perhaps something like:
Here is a list of boards and modules that have been tested by the community:

Comment (itziardelatorre) [reply README.md]: I have added a separated list.

@evedon , @AnttiKauppila : could you please review it again? Thanks!

Comment (itziardelatorre) [reply README.md]: @evedon , @AnttiKauppila : could you please confirm if last update is ok? Thanks!

**Pull request merged**